backend/README.md

```diff
@@ -152,10 +148,10 @@ Also read [CONTRIBUTING.md](https://github.com/fossasia/badgeyay/blob/developmen
 Implementation
 --------------
 
-[generate_badges.py](/app/generate-badges.py) creates svg files from the `csv`, `png` and
+[generate_badges.py](app/generate_badges.py) creates svg files from the `csv`, `png` and
```

Comment:
Please refer to valid location i.e use the location of scripts present in back-end folder.

Reply:
I didn't get this. generate_badges.py is present in backend only in the app folder

Reply:
@rupav

Reply:
I mean to write `backend/app/generate_badges.py` as a path, I hope now you got it 👍

Reply:
@ParthS007 But sir I am fixing the links of README.md present in the backend folder!

Reply:
Doing this backend/app/generate_badges.py would not work I think.

Reply:
If backend needs to be included in the path then I can change it with (../backend/app/generate_badges.py)

Reply:
ohh , my mistake then its LGTM 👍

Reply:
Okay thanks for the time 👍